### DISCUSSION TITLE
I think I fried my scooter :(

### DISCUSSION THREAD

**Discussion by stevenperich** [timeylies/How-to-convert-a-Ninebot-SNSC-2.2-or-2.3-Scooter#4] [Q&A] [answered]
OK so I was removing the ESC and sliced out some plastic/rubbery stuff the ESC is in.  After removing what I thought would be enough to expose the 3 pads, I carefully plugged everything back in again - but outside the frame. So all 3 Julet connectors and the large (drive?) cable and the battery connection.  When I went to power it on unfortunately the dashboard went all crazy for a second or so, and so did the headlight. Now it won't power on (even re-assembled).  Have I fried my scooter? I can't see why.  I noticed if I keep the power button pressed the brake light flashes 5 times quickly then slowly... Does that have any meaning? :(

**Comment by timeylies** [comment]
I never had this kind of experience before so I can’t say much about it, I don’t think it’s fried, since the dash does turn on, you could try moving the scooter while turning it on to see if the dash at least works

If it doesn’t stay on and just turns off maybe your connections aren’t right

**Comment by stevenperich** [comment]
Hi there, nah the dash doesn't turn on (nothing appears on the display), it just starts flashing the brake light at the back in a morse code style of pattern... 5 fast times, 5 slow times, when I plug in the battery and then hold down the power button for a few seconds.  Very weird.

**Reply by timeylies** [reply]
Means that the controller is still on rental firmware, it is trying to say error 55, which means that the battery cover is open. Try flashing the controller with drv126

**Reply by stevenperich** [reply]
very interesting.  I did wonder if this was it just trying to flash error codes at me.  Obviously I was testing with the battery cover open.  I could try again with it closed up and see if I just get error 10...

**Reply by timeylies** [reply]
Good idea

**Reply by stevenperich** [reply]
ok, well, I just tried flashing the ESC..no dice. Can't connect to it no matter what - tried breadboard pins, using the 5VDC line, even soldering the pins in and no cigar. When I tell STM32 to connect the STLink v2's LED flashes a few times then I get "Cannot connect to the target! Please select Connect Under Reset mode from Target > Settings menu and try again". Have tried that and tried enabling readout protection too.  I guess this thing is toasted.

**Reply by timeylies** [reply]
Your supposed to disable readout protection and then hit apply

**Reply by stevenperich** [reply]
I tried, when I went into target > option bytes it just says the same thing. No menu to disable readout protection on.
I’ve noticed that when the stlink is connected to the pins the leds flash but they don’t when I rip the jumpers off so it’s doing sommmeething… just not giving me disable readout protection

**Reply by stevenperich** [reply]
I’ve also tried hitting connect in different scenarios to eliminate wiring issues. If I have all 3 pins connected the red and blue LEDs flash on the STLink when I connect, but if I disconnect ANY of the 3 pins then try to connect, no flashing. So that to me at least rules out an issue with connectivity on any of those 3 pins.

**Reply by timeylies** [reply]
also, make sure the option bytes are all normal, since it won’t work with connect under reset or any other mode

**Reply by timeylies** [reply]
try checking the serial number on the esc, if it starts with AT, it isn’t an stm32 esc, and we’ll have to use openocd to flash it

**Reply by stevenperich** [reply]
Aah there you go. It starts with AT. I just followed the instructions over at scooterhacking to download and use ReFlasher and it worked with these settings:

Scooter: Ninebot Max G30
peripheral: DRV
Serial Number: N4GSD0000C000
Distance: 0km
AT32: checked
copy CPUID: unchecked
activate ESC: checked. 

Are these settings ok? Also do you have discord or something ? :)

**Reply by timeylies** [reply, marked as answer]
yep, and i do, discord is timeylies

**Reply by stevenperich** [reply]
Hey there. Messaged you on discord...any tricks to getting the battery out of the case? It's stuck hard in there. is there a good lever point to get it with a screwdriver or something?

**Reply by isakgg617-beep** [reply]
Hello, Stephen! I have the same problem, the serial number of the controller starts with AT, were you able to flash the controller? I tried through the st link, and through reflasher and through openocd, but everywhere it said couldn't connect to target, please help!

**Reply by isakgg617-beep** [reply]
timeylies, Please accept the application as a friend in the discord

**Reply by booopako** [reply]
> yep, and i do, discord is timeylies

Ive got a question my friends dad works at Dott and he has the Spott app its a fleet app for dott could you slide me the 1.2 apk and ill try to do smth so i can change the key

**Comment by stevenperich** [comment]
possibly similar to this.  https://www.reddit.com/r/NinebotMAX/comments/vqr4g4/tail_lights_blinking_as_scooter_wont_turn_on/ maybe i'll check the connections yet again..

**Comment by stevenperich** [comment]
Have checked and rechecked all cables. Nothing amiss. Here’s a video of what happens: https://imgur.com/a/tqoZg2J 
Guess it’s a goner

**Reply by timeylies** [reply]
In that last part of the video, it was flashing error 10, which means dash communication error, so I do think there’s something wrong with the connections. Don’t give up on it just yet.

**Reply by stevenperich** [reply]
given that the first time I reconnected the dash and applied power, the headlight and all the segments on the dashboard display lit up and went crazy for a second or so, it might be possible that I reconnected the dashboard incorrectly (can the Julet cables go the wrong way? I can't see how that's possible?) and that fried it.  Wonder if I should order a new dashboard and see if that works....

**Reply by timeylies** [reply]
Yeah I think you did connect it incorrectly

**Reply by stevenperich** [reply]
Bummer. Have closed it up and got only error 10 and have ordered a new dashboard. fingers crossed. I guess at least I won't need to flash the dash with BLE110 now :)

**Reply by timeylies** [reply]
I hope nothing happens to that new dash, and also, check the dash power wires with a multimeter

**Reply by stevenperich** [reply]
thanks. fingers crossed.  Do you mean check them for resistance? or voltage on a certain pin?

**Reply by timeylies** [reply]
Check the voltage coming out the red and black wires on the dash wires when the controller is plugged into the battery

**Reply by stevenperich** [reply]
yep, getting 5VDC across the red/black wires

**Reply by timeylies** [reply]
Ok nice, maybe that just means that the dash wasn’t working and maybe decided to disable itself or something
